appeals/web/src/server/appeals/appeal-details/site-ownership/__tests__/__snapshots__/site-ownership.test.js.snap

```diff
@@ -0,0 +1,32 @@
+// Jest Snapshot v1, https://goo.gl/fbAQLP
+
+exports[`site-ownership GET /change should render changeSiteOwnership page 1`] = `
```

Comment:
It's not important but I feel it's preferable to use prose rather than variable names etc. in test descriptions